loadtest/control/utils.go

```diff
@@ -57,15 +57,35 @@ func RandomizeUserName(name string) string {
 }
 
 func emulateUserTyping(t string, cb func(term string) UserActionResponse) UserActionResponse {
+	typingSpeed := time.Duration(100+rand.Intn(20)*10) * time.Millisecond // 100-300ms
+
+	typeCorrect := time.NewTicker(typingSpeed)
+	defer typeCorrect.Stop()
+	typeWrong := time.NewTicker(time.Duration(2+rand.Intn(8)) * typingSpeed)
+	defer typeWrong.Stop()
+
 	runes := []rune(t)
 	var term string
 	var resp UserActionResponse
```

Comment:
I don't see a need to declare `resp` above here. We can just do `resp := ` directly inside the loop for each case.

Reply:
It should be assigned to previously stated variable because we are returning at the end of the loop.

Reply:
Oops missed that. Thanks.